### DISCUSSION TITLE
Blender AgX to Affinity Photo

### DISCUSSION THREAD

**Discussion by DjangoMango666** [sobotka/AgX#10] [Q&A]
Hello!
Since I can't find any information on the internet about how to import the AgX Workflow to Affinity I wanted to ask if there is a solution to that? Tried it myself with the same workflow as filmic (different cuts and config of course) but it doesn't work.

Maybe someone out there has some tips or figured it out?

Thanks and greetings, 
djangomango

**Comment by sobotka** [comment]
AgX was a pre-alpha proof of principle. Despite this experimental stage, many folks have been experimenting with the concept, and some have generated alternative configurations off of this pre-alpha state. Affinity seems to be migrating to OpenColorIO version 2.+, but has not yet, and as such, the AgX proof of principle for OpenColorIO version 2.+ will not work.

You may wish to examine https://github.com/MrLixm/AgXc as they have managed to hack an OpenColorIO version 1 configuration together.

**Comment by MrLixm** [comment]
Hey thanks for sharing Troy, yes Affinity is still, unfortunately, working with OCIO v1 which is an issue I try to address in my version.

**Comment by DjangoMango666** [comment]
Hello Troy and Liam!

Thanks for your answers and support! Since I´m not a nearly an expert in this field, I understand now a little bit more of the problem! I installed your branch Liam, and since it’s OCIO v1 compatible I tried it with affinity (all variations) but wasn’t successful. Is there a special way of applying the OCIO adjustment Layer and LUT Layer in affinity? (ICC Activated) 
Also: do you know how to deal with DaVinci?

I know you’re working on it and time is limited! But It would be such an achievement to at least bring it to affinity (who also did a successful filmic implementation). People would jump on it for sure!

Thank you both very much and great work!! For me as someone, who is seeking to mimic a real Photostudio and its PostProduction, it’s a dream with what you came up!

Greetings,
Erwin

**Comment by MrLixm** [comment]
Hey, recorded a quick video of how you can use it in Affinity photo :
https://youtu.be/Sko8gOKgyPs

(Nice that you asked this cause it allowed me to see the new update of the config I planned, fix an issue. I have pushed the fix so please re-download the config.)

**Comment by DjangoMango666** [comment]
Hi,

The video explanation was great and I followed it with my test renderings. At first it worked when I used only white light, but as soon as I turn the light into warmer area the picture breaks and color reproduction doesn’t seem correct. Maybe I made a mistake but I did as you told me so.

Thank you for your help and the video!
Excited to hear what you have to say to my tests :)

Greetings, 
Erwin

Here the Screenshots of my tests:
https://we.tl/t-lI5U1UmVdm￼

**Comment by MrLixm** [comment]
Hey again, do not put the OCIO Transfer that go from `ACES2065-1` to `Linear sRGB` if your input image is not encoded in ACES2065-1.
If you rendered it in Blender using the same config this means the render is probably already encoded as `Linear sRGB` and doesn't need any conversion.

**Reply by DjangoMango666** [reply]
Hello! Sorry for the late reply, been very busy! I have incorporated your answer into my workflow and It worked like a charm! thank you very much!

**Comment by DjangoMango666** [comment]
Hi Liam and Troy!
I guess AgX is really taking off now and a lot of people are realising what great job you did with the AgX Profile! I´m a lecturer for Blender at University myself (specialised on Industrial Design and Product Rendering, where we welcome every bit that takes us further to Photorealism) and with this Semester I would like to switch to AgX and teach the students how to implement it into a production workflow. There's only one problem (since the OCIO V1 got solved by Liam) left, to fully implement it into a production workflow and that is the missing of the false color to check the exposure. Would it be a lot of work to add a false color profile to the AgX config? It's a really important tool in filmmaking, animation and photography. Maybe we can even emulate some real False Color Profiles from real Products like the "Small HD Cine 7", which is a little bit different to the false color from blender (where black is the lowest). Looking forward to your answer(s)!

Greetings, Erwin

**Reply by sobotka** [reply]
False colour didn’t really exist in Blender until I added it, and then folks seemed to arrive at it being a showstopper for not having it. 🤣 

At any rate, a false colour should be easy to design. Somewhere in my repos I even included a Blender node setup to generate one.

Right now I’ve been experimenting with trying to get an AgX out using camera native log encodings for ARRI etc. via DCTLs, as well as finally finish up the Guard Rail.

It shouldn’t take much effort for someone to manage to get a False Colour working for the Configuration Generator working.

**Reply by EaryChow** [reply]
I want to mention my version already includes a false color done post view with a 3x1D LUT.

**Reply by DjangoMango666** [reply]
Oh I didn't know it was you! But who else should it have been, if not you Troy 😄
ok so before I could answer you I had to look up what DCTLs and Guard Rails are, but I´m trying to learn here (still don't know what Guard Rails are) :D So as far as I understand, the plan for the future is to be able to save LogC Footage out of Blender which can then be transformed in Davinci to BT709 or whatever (via DCTL?)? Do you also have Affinity in mind while developing AgX (since it is only using OCIOV1) or you hope that they will just upgrade to OCIO V2 finally?

If I would know how to code and understand what you're doing I`d offer my help, but I´m from the User-Side. So all I can do is communicate it to the community or something similar, so you don't have to deal with 1000 people like me 🙃

@EaryChow I just checked out your fork of AgX! Nice Work, also with the "Looks". Is there any chance to fuse your work with the Fork from MrLixm (AgXc), so it's also usable with Affinity or any other OCIO V1 Software?

Greetings!
